includes/discovery/mempools/extreme.inc.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+if ($device['os'] == 'xos') {
+    echo 'EXTREME-SOFTWARE-MONITOR-MIB';
+
+    $total = str_replace('"', "", snmp_get($device, "1.3.6.1.4.1.1916.1.32.2.2.1.2.1", '-OvQ'));
```

Comment:
In this script it's only used to validate that the $total and $avail values are valid so the polling will generate proper values. A custom poller is made as well where $total and $avail are used to populate the $mempool array.